src/Helpers.cpp

```diff
@@ -182,8 +182,8 @@ void LaunchpadBase::sendMessage(unsigned int first_byte, ...) {
 	if (isConnected() == false) return;
 	va_list varlist;
 	va_start(varlist, first_byte);
-	unsigned char byte = first_byte;
-	while (byte != MIDI_MESSAGE_SYSEX_END || byte < 0 || byte > 255) {
+	unsigned int byte = first_byte;
+	while (byte != MIDI_MESSAGE_SYSEX_END  && byte != MIDI_MESSAGE_END && byte >= 0 && byte <= 255) {
```

Comment:
We can remove the `&& byte != MIDI_MESSAGE_END` since it's defined as -1, which is lower than 0 and will fail the `byte >= 0` check, and remove one less definition.  😋